src/app/core/services/data.service.ts

```diff
@@ -211,7 +211,8 @@ export class DataService {
    */
   private csvParser(csvText: string): [{}] {
     return parse(csvText, {
-      columns: true
+      columns: true,
+      skip_empty_lines: true
```

Comment:
is the `trim` option needed to remove leading / trailing whitespace?

Reply:
Seems like another possible source of error, I guess we can add it in